core/src/main/kotlin/org/evomaster/core/problem/rest/service/fitness/AbstractRestFitness.kt

```diff
@@ -756,6 +761,12 @@ abstract class AbstractRestFitness : HttpWsFitness<RestIndividual>() {
             responseClassifier.updateModel(a, rcr)
         }
 
+        if (config.security && config.ssrf) {
+            if (SSRFAnalyser.hasVulnerableInputs(a)) {
+                rcr.setVulnerableForSSRF(true)
```

Comment:
is this correct?  you are check if there is any malicious payload in the action, not that, once action is executed, then a call was made to WM, isn't it?